src/meta/src/hummock/compaction/picker/base_level_compaction_picker.rs

```diff
@@ -166,7 +167,9 @@ impl LevelCompactionPicker {
             self.config.level0_max_compact_file_number,
             overlap_strategy.clone(),
             self.developer_config.enable_check_task_level_overlap,
-            self.config.max_l0_compact_level_count as usize,
+            self.config
+                .max_l0_compact_level_count
+                .unwrap_or(compaction_config::max_l0_compact_level_count()) as usize,
```

Comment:
reminder: `compaction_config::max_l0_compact_level_count()` makes sense only because it's not configurable in toml config file. Otherwise we should use the value from config file rather than the static default one.

Was the omission of max_l0_compact_level_count from the config file unintentional?

Reply:
Its not omission, we change the setting of `max_l0_compact_level_count ` to `Option` and the default value is `None`  (So it's invisible.)

Reply:
I mean it's not included [here](https://github.com/risingwavelabs/risingwave/blob/96de66440d5f8238d920ac76ce04cbd6247bcb21/src/common/src/config.rs#L2300)